app/main.py

```diff
@@ -2,0 +1,40 @@
+from __future__ import annotations
+
+
+class Animal:
+    alive: list = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
```

Comment:
Clean Code: It's a good practice to add comments to your code. It would be helpful to add a comment here explaining why you're appending `self` to the `alive` list.